backend/src/helpers/students.js

```diff
@@ -1,9 +1,22 @@
 const db = require('../models/student')
 // const debug = require('debug')('app:students')
 
-exports.getAllStudents = async function () {
+const defaultOptions = {
+  filters: {}
+}
+exports.getAllStudents = async function (options = defaultOptions) {
+  const filterStatus = options.filters.status
+
+  let query = db.Student.query()
+
+  if (filterStatus) {
+    query = query
+      .withGraphJoined('status')
+      .where('status.Description', filterStatus)
```

Comment:
```suggestion
      .where('status.Description', 'ilike', filterStatus)
```

Hi Laura, suggest to make it case-insensitive :)